src/components/account/EditParkingSpotsModal.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+const EditParkingSpotsModal = ({
+  valetData,
+  setParkingSpots,
+  handleParkingSpotsUpdate,
+}) => {
+  return (
+    <dialog id="editParkingSpots" className="modal">
+      <div className="modal-box">
+        <form
+          method="dialog"
+          className="flex flex-col items-center"
+          onSubmit={(e) => {
+            e.preventDefault();
+            handleParkingSpotsUpdate();
+          }}
+        >
+          <label className="form-control w-full max-w-xs">
+            <div className="label">
+              <span className="label-text">Edit Number of Parking Spots</span>
+            </div>
+            <input
+              type="text"
```

Comment:
I think this input element could be type="number" to easily prevent unexpected values

Reply:
Good catch, change it to number type